projects/components/src/button/button.component.ts

```diff
@@ -10,7 +10,7 @@ import { ButtonRole, ButtonSize, ButtonStyle } from './button';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <ht-event-blocker event="click" class="button-container" [enabled]="this.disabled">
-      <button class="button" [ngClass]="this.getStyleClasses()">
+      <button class="button" [ngClass]="this.getStyleClasses()" [htTrack] [htTrackLabel]="this.label">
```

Comment:
do we capture something like the route path or url here? Seems like it's worth not just tracking "Submit" for example, but where that submit is.

Reply:
Yes. URL change is tracked via router events and it can be enabled via `enablePageTracking` option

Reply:
but those are separate events. If I'm looking at a click event for a button labeled "submit" how do I know what page it's on? Or do the tools provide that by tracking the event timeline?

Reply:
These are the things we can capture and add to the "eventData" object from inside the telemetryImpl service. But AFAIU, the telemetry tools will timestamp these anyway, so we can connect the dots there. Let's iterate over this.